tests/tools/test_storage.py

```diff
@@ -139,81 +135,65 @@ async def test_list_block_devices_lsblk_success(
         mocker.patch(
             "linux_mcp_server.tools.storage.execute_command",
             AsyncMock(return_value=(1, "", "command failed")),
         )
 
-        result = await mcp.call_tool("list_block_devices", {})
-        assert isinstance(result, tuple)
-        assert len(result) == 2
-        assert isinstance(result[0], list)
-        output = result[0][0].text
+        result = await mcp_client.call_tool("list_block_devices", {})
+        result_text = result.content[0].text
 
-        assert "Error" in output or "Unable" in output
+        assert "Error" in result_text or "Unable" in result_text
 
-    async def test_list_block_devices_file_not_found(self, mocker):
+    async def test_list_block_devices_file_not_found(self, mocker, mcp_client):
         """Test list_block_devices when lsblk is not available."""
         mocker.patch("linux_mcp_server.tools.storage.execute_command", side_effect=FileNotFoundError("lsblk not found"))
 
-        result = await mcp.call_tool("list_block_devices", {})
-        assert isinstance(result, tuple)
-        assert len(result) == 2
-        assert isinstance(result[0], list)
-        output = result[0][0].text
+        result = await mcp_client.call_tool("list_block_devices", {})
+        result_text = result.content[0].text
 
-        assert "Error" in output or "not found" in output
+        assert "Error" in result_text or "not found" in result_text
 
-    async def test_list_block_devices_remote_execution(self, mock_storage_execute_command):
+    async def test_list_block_devices_remote_execution(self, mock_storage_execute_command, mcp_client):
         """Test list_block_devices with remote execution."""
         mock_storage_execute_command.return_value = (0, "NAME   SIZE TYPE\nsda    1TB  disk", "")
 
-        result = await mcp.call_tool("list_block_devices", {"host": "remote.host.com"})
-        output = verify_result_structure(result)
-
-        assert "=== Block Devices ===" in output
-        assert "sda" in output
-        assert "=== Disk I/O Statistics" not in output
+        result = await mcp_client.call_tool("list_block_devices", {"host": "remote.host.com"})
+        result_text = result.content[0].text
 
-        # Verify execute_command was called with host parameter
-        mock_storage_execute_command.assert_called_once()
-        call_kwargs = mock_storage_execute_command.call_args[1]
-        assert call_kwargs["host"] == "remote.host.com"
+        assert "=== Block Devices ===" in result_text
+        assert "sda" in result_text
+        assert "=== Disk I/O Statistics" not in result_text
+        assert mock_storage_execute_command.call_count == 1
+        assert mock_storage_execute_command.call_args.kwargs["host"] == "remote.host.com"
 
-    async def test_list_block_devices_exception_handling(self, mock_storage_execute_command):
+    async def test_list_block_devices_exception_handling(self, mock_storage_execute_command, mcp_client):
```

Comment:
Can probably be grouped  with https://github.com/rhel-lightspeed/linux-mcp-server/pull/69/changes#r2624204574